### PR TITLE
IDEX-3360: Rebuild dev machine images and add them to the list of pre-defined recipes

### PR DESCRIPTION
**Pull request by ghost**
**_1 Upvote**_

### CODE REVIEW

**Comment by garagatyi** [comment]
You said that you created images fromubuntu, debian and centos. I can't find centos recipe here.

File: assembly-sdk-war/src/main/resources/predefined-recipes.json [hunk 1]
```diff
@@ -3,31 +3,9 @@
...
-      "users": {}
-    },
-    "id": "recipe_jdk8_tomcat8_git",
-    "type": "docker",
-    "script": "FROM codenvy/jdk8_maven3_tomcat8"
-  },
-  {
-    "creator": "codenvy",
-    "tags": [
       "ubuntu"
```
Comment (garagatyi) [inline assembly-sdk-war/src/main/resources/predefined-recipes.json]: Looks like we need additional tags here: maven, git, curl, nmap, etc.

File: assembly-sdk-war/src/main/resources/predefined-recipes.json [hunk 2]
```diff
@@ -42,15 +20,15 @@
...
-    "script": "FROM ubuntu\nCMD tail -f /dev/null"
+    "script": "FROM codenvy/ubuntu_jdk8"
   },
   {
     "creator": "codenvy",
     "tags": [
-      "java",
-      "busybox"
+      "debian",
+      "java"
```
Comment (garagatyi) [inline assembly-sdk-war/src/main/resources/predefined-recipes.json]: Please, add additional tags

**Comment by ghost** [comment]
No centos for now. Maybe later.

**Comment by akorneta** [comment]
:+1:

**Comment by garagatyi** [comment]
LGTM

**Comment by vparfonov** [comment]
ok

**Pull request merged**